specification/metrics/listunitprice.md

```diff
@@ -1,8 +1,8 @@
 # List Unit Price
```

Comment:
Practitioners can calculate this by dividing List Cost to usage amount. Unless I'm missing something this column might be considered redundant. Happy to discuss further this topic with the group.